Makefile

```diff
@@ -6,6 +6,10 @@
 
 all: fmt clippy doc test
 
+static-toml:
+	taplo format --check ./*.toml
+	taplo format --check ./crates/**/*.toml
```

Comment:
Could also use the taplo config file to add included dirs: https://taplo.tamasfe.dev/configuration/file.html but we use `make` so I guess it's no different.